gobblin-iceberg/src/test/resources/publisherTest/data.json

```diff
@@ -1,8 +1,10 @@
-{
-  "id": 1,
-  "name": "Alyssa"
-}
-{
-  "id": 2,
-  "name": "Bob"
-}
\ No newline at end of file
+[
+  {
```

Comment:
Cleaning up "invalid" json syntax. I think the reason this worked before is because of Jackson magic